blog/2024-07-19-coordinator_async_setup.md

```diff
@@ -0,0 +1,43 @@
+---
+author: Josef Zweck
+authorURL: https://github.com/zweckj
+title: Setting up `DataUpdateCoordinators` with `_async_setup`
```

Comment:
Now it could like `DataUpdateCoorindators` is a thing, but it's only without the `s`. Maybe we can call it. New lifecycle hook for coordinators or something like that

Reply:
`DataUpdateCoordinator` now supports a setup method. Or something as such perhaps.

Reply:
Unresolving as the title isn't correct still

Reply:
I did remove the `s` so I resolved it, updated title again